### PR TITLE
Fix: Loader doesn't filter asset list recursively

### PR DESCRIPTION
**Pull request by Tilix4**
## Changelog Description
Since OP 3.15.0 Loader doesn't filter asset list recursively

## Testing notes:
1. Open Loader
2. Try to filter by typing words

### CODE REVIEW

File: openpype/tools/utils/models.py [hunk 1]
```diff
@@ -235,6 +240,7 @@ def filterAcceptsRow(self, row, parent_index):
         )
 
 
+
```
Comment (BigRoy) [inline openpype/tools/utils/models.py:243]:
```suggestion
```

**Comment by iLLiCiTiT** [comment]
I found out it's broken on other places too. Would it be ok if I would create new PR?

**Comment by Tilix4** [comment]
> I found out it's broken on other places too. Would it be ok if I would create new PR?

@iLLiCiTiT, sure, the goal is for it to work, no matter who does it :)

**Comment by iLLiCiTiT** [comment]
Made PR fixing also other places https://github.com/ynput/OpenPype/pull/4597 @Tilix4 thanks for spotting and reporting the issue.

**Pull request closed without merging**